include/dali/c_api.h

```diff
@@ -335,6 +335,11 @@ DLL_PUBLIC size_t daliMaxDimTensors(daliPipelineHandle *pipe_handle, int n);
  */
 DLL_PUBLIC unsigned daliGetNumOutput(daliPipelineHandle *pipe_handle);
 
+/**
+ * @brief Returns a string indicating name of the output given by id
```

Comment:
Does it make sense to mention that the returned pointer is valid as long as the pipeline exists?